packages/server/src/service/trpc.test.ts

```diff
@@ -44,10 +46,7 @@ test('trpc.register uses e164 number on sendFaxes', async () => {
       remoteAddress: '127.0.0.1',
     }
   })
-  const res = await trpc.register(stateInfo, { uid: 'abcdefghij' }, req)
-  // Ensures res.data is processed
-  const cont = res?.continuation as () => Promise<void>
-  await cont()
+  await trpc.register(stateInfo, { uid: 'abcdefghij' }, req)
```

Comment:
We want to make sure the continuation is run right?

Reply:
Because we no longer return a function at `trpc.register` we don't need this block of code anymore, more details at #172 (sorry for being unclear about this)